data/persons.txt

```diff
@@ -854,37 +854,35 @@ person "Prototype B3-CC4"
 	frequency 20000
 	government "Author"
 	personality
-		forbearing mining surveillance unconstrained
+		forbearing mining surveillance unconstrained swarming
```

Comment:
Surveillance and swarming don't really mix

Reply:
Given that it already has mining, I'll remove the tactical and asteroid scan power as well as the surveillance personality